### DISCUSSION TITLE
show / hide card

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#2496] [Help]
Is it possible to show or hide a card from the layout depending on the selection of an option from a select?

**Comment by OmarASharaf** [comment]
Yes you can, by using the method reactive() on the select and then use the "hidden()" or "visible()" method on the Card and use the Closure to check the chosen value of the select

```
Select::make('type')->options([
      'normal' => "normal",
      'easy' => "easy",
]) ->reactive()

Card::make()->schema([
   TextInput::make('name')->required()          
])->hidden(fn (Closure $get) => ($get('type') == 'normal')),
```

https://filamentphp.com/docs/2.x/forms/advanced#reloading-the-form-when-a-field-is-updated
https://filamentphp.com/docs/2.x/forms/advanced#dependant-fields--components

**Reply by jcgmalave** [reply]
Thanks a lot!!!
